capstone/backend/src/main/java/com/google/servlets/GetUserListsServlet.java

```diff
@@ -0,0 +1,63 @@
+package com.google.servlets;
+
+import com.google.gson.Gson; 
+import java.lang.StringBuilder;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/api/v1/get-user-lists")
+public class GetUserListsServlet extends HttpServlet {
+  private class RequestBody {
+    int userId;
+  }
+
+  private class ResponseBody {
+    List<List<String>> userLists;
+
+    public ResponseBody(List<List<String>> userLists) {
```

Comment:
Seems like when you create a list [in your previous servlet here](https://github.com/googleinterns/step39-2020/blob/fbfeac1198c95519cd4ce1e92854521c5e6bb98e/capstone/backend/src/main/java/com/google/servlets/CreateOrUpdateUserlistServlet.java#L31), you use the `UserList` object that also has a display name and listId. So this servlet should also include those information as well - the types should be consistent. (Maybe take out UserList definition and put it in a separate common file.)

(Also this is probably better as a `GET` request, since you're not mutating data.